docs/source/operations/baf_verify.md

```diff
@@ -238,6 +238,41 @@ For final checking of the validity of the indy network.
 
 Upon successful connection, should display a `Pool Connected Successfully` msg.
 
+
+---  
+
+
+## R3 Corda Checks
+
+The flow chart shows the R3 Corda process. To verify the steps of deployment, follow the Verification Table 'R', to troubleshoot the general errors.
+
+![](./../_static/corda_flowchart.png)
+
+----
+### R3 Corda Troubleshooting
+### Table 'N'
+
+|Section|Sub-Section|Problem|Possible Cause|Solution|
+|-------|-----------|-------|--------------|--------|
+| R1 | a | **Ansible playbook failed** <br>Playbook execution terminated at <br>**Role:** create/certificates/ambassador <br>**Task:** Copy generated ambassador tls certs to given build location. <br>**Error:** Destination directory, example: /home/[user]/build/corda/doorman/tls, does not exist | Folder to copy tls certs does not exist. | **network.network_services.certificate** value is either misspelled or directory doesn't exist. |

```

Comment:
If this is open-source Corda, still following items are missing from the flowchart

1. Notary registration failure
2. Notary DB failure
3. Node DB Failure.

Also, the flowchart talks about R1-R5, but in the table here, there is only R1 and R2. What is a user supposed to do when they go to R5?